### ISSUE TITLE
Commented out array elements break indentation

### ISSUE THREAD

**Issue by flaviovs** [emacs-php/php-mode#726]
Correctly indented by *php-mode*:
```php
<?php
$a = [
    'aaa' => [
        'bee' => 2,
    ],
    'foo' => [
         'bar' => 1,
    ],
    'lee' => 2,
];
```

Comment out an array element:

```php
<?php
$a = [
    'aaa' => [
        'bee' => 2,
    ],
    // 'foo' => [
    //      'bar' => 1,
    // ],
    'lee' => 2,
];
```

Request auto-indentation (i.e., <kbd>Tab</kbd> over lines, `indent-region`, etc.):
```php
<?php
$a = [
    'aaa' => [
        'bee' => 2,
    ],
    // 'foo' => [
        //      'bar' => 1,
        // ],
        'lee' => 2,
];
```

* php-mode 20230103.1226
* `(php-enable-psr2-coding-style)`

**Comment by zonuexe** [comment]
@flaviovs Thank you for reporting, maybe fixed in #732.

**Comment by flaviovs** [comment]
Hi @zonuexe,

I confirm, after upgrading to latest on MELPA (20230305.2120) the issue is gone.

Thank you
